### DISCUSSION TITLE
How to disable spell checking of comments?

### DISCUSSION THREAD

**Discussion by elwood68** [streetsidesoftware/vscode-spell-checker#888] [Q&A]
I don't 'care' about the spelling of comments. It would be great to have a setting to ignore commented lines.
thanks.

**Comment by Jason3S** [comment]
It is possible, but not obvious.

There are a few [predefined patterns](https://github.com/streetsidesoftware/vscode-spell-checker#predefined-regexp-expressions) than can be used to exclude or include items to be checked.

## Using VS Code Settings:

This is the same as `.vscode/settings.json` below.
![image](https://user-images.githubusercontent.com/3740137/119217623-b1e34d80-badb-11eb-86e1-1d1984afb077.png)

Becomes
![image](https://user-images.githubusercontent.com/3740137/119217691-19010200-badc-11eb-9ee5-4de12129fd2b.png)


## `cspell.json`

Add a section to `cspell.json`

```js
"ignoreRegExpList": [
  "CStyleComment"
]
```

## `.vscode/settings.json` or `*.code-workspace`

In a `*.code-workspace` file, it must be nested within the `settings` section.

```js
"cSpell.ignoreRegExpList": [
  "CStyleComment"
]
```
